File: String/RemoveOutermostParenthesis-1021/code.cpp
class Solution {
public:
    string removeOuterParentheses(string s) {
        string result="";
        int open = 0;
        for(int i = 0 ;i< s.length();i++){
            if(s[i] == '('){
                if(open > 0) result += '(';
                open++;
            }else{
                open--; 
                if(open > 0){
                    result += ')';
                }
                
            }
        }
        return result;
    }
};

File: Binary Tree/PreorderTraversal-144/code.py
# Definition for a binary tree node.
# class TreeNode:
#     def __init__(self, val=0, left=None, right=None):
#         self.val = val
#         self.left = left
#         self.right = right
class Solution:
    def preorder_traversal(self, node, result):
        if node is None:
            return
        result.append(node.val)
        self.preorder_traversal(node.left, result)
        self.preorder_traversal(node.right, result)

    def preorderTraversal(self, root: Optional[TreeNode]) -> List[int]:
        result = []
        self.preorder_traversal(root, result)
        return result